.github/workflows/CI.yml

```diff
@@ -25,4 +25,4 @@ jobs:
         run: npm run lint:check
 
       - name: Run tests
-        run: forge test
+        run: forge test --fork-url https://eth-mainnet.g.alchemy.com/v2/jf8PEqEElc58ulLAU_R2KO3UnTtmMJZH --fork-block-number 14563292
```

Comment:
would probably want to set this up on the repo secrets

Reply:
true, I just don't have access to that part of the repo and I created some dummy api key to go around it